src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -362,6 +362,9 @@ def remove_snapshot(self, snapname):
         rmsnap(self.fs, snappath)
 
     def snapshot_info(self, snapname):
+        if inherited_snap(snapname):
+            raise VolumeException(-errno.EINVAL,
```

Comment:
Operations on inherited snaps return EINVAL(like snap create, rm), hence followed the same here though ENOENT also fits well here.

Reply:
maybe it's safer to give error message that 'snapshot name is invalid'?